### PR TITLE
Fixes #8786: Broken technique tests

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/8786

### CODE REVIEW

File: techniques/system/inventory/1.0/fusionAgent.st [hunk 1]
```diff
@@ -60,9 +60,8 @@ bundle agent doInventory
   methods:
     ignore_disable_inventory|!disable_inventory::
       "any" usebundle => doInventory_always;
-  reports:
     !(ignore_disable_inventory|!disable_inventory)::
-      "INFO: The file ${g.rudder_disable_inventory_file} is present. Rudder will not send an inventory. Use rudder agent inventory --force to override this temporarily";
+      "any" usebundle => rudder_common_report("Inventory", "log_info", "&TRACKINGKEY&", "inventory", "None", "The file ${g.rudder_disable_inventory_file} is present, so Rudder will not send an inventory. Use 'rudder agent inventory -f' to override this temporarily");
```
Comment (ncharles) [inline techniques/system/inventory/1.0/fusionAgent.st]: shouldn't it go also in initial promises ?

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

File: techniques/system/inventory/1.0/fusionAgent.st (hunk 1 above)
Comment (amousset) [inline techniques/system/inventory/1.0/fusionAgent.st]: Indeed! I also made it a report in initial promises and updated the message.

**Comment by amousset** [comment]
Commit modified

File: techniques/system/inventory/1.0/fusionAgent.st (hunk 1 above)
Comment (jooooooon) [inline techniques/system/inventory/1.0/fusionAgent.st]: Hmmm, this looks like the 2.11 version says "--force". Could you make a separate PR to correct that to "-f"?

Comment (jooooooon) [inline techniques/system/inventory/1.0/fusionAgent.st]: Hang on, though. I don't think we want this to be a Rudder report. The goal here is that when you run the "rudder agent inventory" command, you get a human readable message telling you why the inventory wasn't made. Will this work OK with the Rudder report style?

Comment (amousset) [inline techniques/system/inventory/1.0/fusionAgent.st]: The change was made in 3.1 (https://github.com/Normation/rudder-techniques/pull/996).

Comment (jooooooon) [inline techniques/system/inventory/1.0/fusionAgent.st]: Ah, right. Of course :)

Comment (amousset) [inline techniques/system/inventory/1.0/fusionAgent.st]: There is already a check in the "rudder agent inventory" command before starting CFEngine to get a clearer error message. I changed the message from info to warn to make it visible in a normal run.

Comment (jooooooon) [inline techniques/system/inventory/1.0/fusionAgent.st]: OK, great!

**Comment by amousset** [comment]
Commit modified
